pageserver/src/http/openapi_spec.yml

```diff
@@ -569,7 +569,17 @@ paths:
               schema:
                 $ref: "#/components/schemas/NotFoundError"
         "409":
```

Comment:
Above there is a
```yaml
        "404":
          description: Timeline not found
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/NotFoundError"
```
which doesn't look as a valid response in this case, right?

I'm not sure that 404 is a valid response for this API handler at all

Reply:
Yeah, most likely copy-pasta, but, will be handled separately.

Reply:
https://github.com/neondatabase/neon/issues/5732